### PR TITLE
Make version in docs update automatically when changed in setup.py

### PR DESCRIPTION
**Pull request by sarah256**

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -6,24 +6,31 @@
...
 # documentation root, use os.path.abspath to make it absolute, like shown here.
-#
-import os
-import sys
+
 sys.path.append(os.path.abspath('../'))
 
 # -- Project information -----------------------------------------------------
 
+log = logging.getLogger('estuary')
```
Comment (mprahl) [inline docs/conf.py]: This can be removed.

File: docs/conf.py [hunk 2]
```diff
@@ -6,24 +6,31 @@
 # full list see the documentation:
 # http://www.sphinx-doc.org/en/master/config
 
+from __future__ import unicode_literals
+
+import os
+import sys
+import logging
```
Comment (mprahl) [inline docs/conf.py]: This can be removed.

File: setup.py [hunk 3]
```diff
@@ -5,6 +5,7 @@
 with open('requirements.txt', 'r') as f:
     requirements = f.readlines()
 
+# Variables need to be referenced elsewhere
```
Comment (mprahl) [inline setup.py]: This can be removed

File: docs/conf.py (hunk 2 above)
Comment (sarah256) [reply docs/conf.py]: i need os and sys for the path setup part, i just moved them upwards to be with the others. the logging i will remove though

Comment (mprahl) [reply docs/conf.py]: Yeah, the comment was just on the `import logging` line.

**Pull request merged**